pkg/fs/sync.go

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) 2019 The virnsnap authors. See file "AUTHORS".
+// Licensed under the MIT License. You have obtained a copy of the License at
+// the "LICENSE" file in this repository.
+
+// Package fs implements helper functions for handling filesystem related
+// tasks.
+package fs
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/joroec/virsnap/pkg/instrument/log"
+)
+
+// Sync is a minimal and opinionated wrapper around a call to
+// "rsync -avp <source> <destination>"
+func Sync(source string, destination string, logger log.Logger) error {
```

Comment:
Not sure if this should be part of this feature. This is quite opinionated, maybe it's better to leave it up to the user how he ship the exports away ?

Reply:
Ah I might stand corrected, are we using `rsync` to copy the saved export into the specified directory? Can't we use `io.Copy` ?

Reply:
We're using `rsync` to compare the current hard disk image with an disk image that may have already been exported at a previous export run. In this case, we do not need to transmit the whole file again, since `rsync` transmits only the (differences)[https://rsync.samba.org/tech_report/node2.html]. `io.Copy` is just a naive copy function and therefore does not allow to make differential file transmissions.